config/irqbalance.conf

```
@@ -0,0 +1 @@
+CONFIG_PACKAGE_irqbalance=y
```

Build it as a module:
```suggestion
CONFIG_PACKAGE_irqbalance=m
```